client/src/components/ProTx/components/dashboard/DashboardDisplay.js

```diff
@@ -48,12 +48,12 @@ function DashboardDisplay() {
     if (mapType === 'maltreatment') {
       // maltreatment only has county data.
       setGeography('county');
-      setUnit('percent');
+      setUnit('count');
```

Comment:
```suggestion
      setUnit('rate_per_100k_under17');
```
https://jira.tacc.utexas.edu/browse/COOKS-238 is rate.  From last sprint we only allow maltreatment is either  percent or rate_per_100k_under17 and then demographics is either percent or count.  That gets configured in DisplaySelectors. 


which would explain this:
> I am encountering an inconsistent rendering issue with the DisplaySelector labelRadioBtn1 not being actively selected in the UI by default, even though the component state is correct and renders the plot using the correct unit on map selection.